apps/condo/domains/resident/schema/FindUnitsByAddressService.test.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * Generated by `createservice resident.FindUnitsByAddressService --type queries`
+ */
+
+const { makeClient, expectToThrowAuthenticationErrorToResult, expectToThrowAccessDeniedErrorToResult } = require('@open-condo/keystone/test.utils')
+
+const { registerNewOrganization } = require('@condo/domains/organization/utils/testSchema')
+const { PARKING_UNIT_TYPE, FLAT_UNIT_TYPE, APARTMENT_UNIT_TYPE, COMMERCIAL_UNIT_TYPE, WAREHOUSE_UNIT_TYPE } = require('@condo/domains/property/constants/common')
+const { buildingMapJson } = require('@condo/domains/property/constants/property')
+const { getUnitsFromSections } = require('@condo/domains/property/utils/serverSchema/helpers')
+const { createTestProperty, updateTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { findUnitsByAddressByTestClient } = require('@condo/domains/resident/utils/testSchema')
+const { makeClientWithResidentUser, makeClientWithSupportUser, makeClientWithServiceUser, makeClientWithStaffUser } = require('@condo/domains/user/utils/testSchema')
+
+let anonymousClient
+let supportClient 
+let residentClient
+let staffClient
+let serviceClient
+
+let propertyWithoutMap
+let propertyWithMap
+let propertyWithMapSectionUnits, propertyWithMapParkingUnits
```

Comment:
This is just a comment, but it seems to me that we define common entities in the body of the test. Actually in most cases inside describe block